dynamite/orblib.py

```diff
@@ -753,4 +755,43 @@ def get_ml_of_original_orblib(self):
         ml_original = float((lines[-9])[0])
         return ml_original
 
+    def read_orbit_property_file(self):
+        """Read the file `*orbclass.out` files
+
+        This file contains angular momenta Lx, Ly, Lz and v_rms of the orbits.
+        Results are stored in and `self.table`
+        
+        """
+        nE = self.settings['nE']
+        nI2 = self.settings['nI2']
+        nI3 = self.settings['nI3']
+        ndither = self.settings['dithering']
+        norb = int(nE*nI2*nI3)
+        ncol = ndither**3
+        nrow = norb
+        orbclass1 = np.genfromtxt(self.mod_dir+'datfil/orblib.dat_orbclass.out')
+        orbclass1 = orbclass1.ravel()
+        orbclass1 = orbclass1.reshape((5, ncol, nrow), order='F')
+        orbclass2 = np.genfromtxt(self.mod_dir+'datfil/orblibbox.dat_orbclass.out')
+        orbclass2 = orbclass2.ravel()
+        orbclass2 = orbclass1.reshape((5, ncol, nrow), order='F')
```

Comment:
I have not tested the code yet, but shouldn't this line 777 read
`orbclass2 = orbclass2.reshape((5, ncol, nrow), order='F')` (orbclass2.reshape instead of orbclass1.reshape)?